### PR TITLE
Issue #2724059 by mglaman, PieterJanPut, bojanz: Cart block quantity …

### PR DESCRIPTION
**Pull request by mglaman**
…text is not translatable

### CODE REVIEW

File: modules/cart/src/Plugin/Block/CartBlock.php [hunk 1]
```diff
@@ -74,7 +75,7 @@ public static function create(ContainerInterface $container, array $configuratio
    */
   public function defaultConfiguration() {
     return [
-      'item_text' => $this->t('items'),
+      'item_text' => 'item' . PluralTranslatableMarkup::DELIMITER . 'items',
```
Comment (mglaman) [inline modules/cart/src/Plugin/Block/CartBlock.php:78]: I'm unsure if this is to be passed into translation here or not.

Comment (mglaman) [inline modules/cart/src/Plugin/Block/CartBlock.php:78]: Went to, and got code violation warnings for concatenating inside of a translate function. There's no core pattern here [that I can see] to follow.

Comment (mglaman) [inline modules/cart/src/Plugin/Block/CartBlock.php:78]: @bojanz I asked around, there is no standard way, especially with config. I think this is good to go.

**Comment by bojanz** [comment]
No longer relevant. See https://www.drupal.org/node/2724059#comment-11538119

**Pull request closed without merging**